Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -49,6 +49,10 @@ class StoreKit1IntegrationTests: BaseBackendIntegrationTests {
         return .disabled
     }
 
+    func testIsSandbox() {
```

Comment:
I moved this here so it runs in SK1 and SK2 separately, in case that logic ever diverges (like in iOS 16 potentially?)